tox.ini

```diff
@@ -16,9 +16,16 @@ setenv=
 
 basepython=
     py36: python3.6
+    py38: python3.8
 
 deps=
     dj111: Django>=1.11,<1.12
+    dj20: Django>=2.0,<2.1
+    dj21: Django>=2.1,<2.2
+    dj22: Django>=2.2,<2.3
+    wag113: wagtail>=1.13,<1.14
+    wag23: wagtail>=2.3,<2.4
+    wag28: wagtail>=2.8,<2.9
```

Comment:
Since we're specifying Wagtail versions to install here, we can remove [the line that installs cfgov-refresh's pinned version of Wagtail](https://github.com/cfpb/teachers-digital-platform/blob/master/install_cfgov_refresh.sh#L5). Otherwise, the cfgov-refresh pinned version of Wagtail will always override this.